src/Symfony/Component/Notifier/Bridge/FreeMobile/FreeMobileTransport.php

```diff
@@ -60,7 +60,7 @@ protected function doSend(MessageInterface $message): SentMessage
         $endpoint = sprintf('https://%s', $this->getEndpoint());
 
         $response = $this->client->request('POST', $endpoint, [
-            'json' => [
+            'query' => [
```

Comment:
can you please try using `body` here instead?
if we keep `query`, can we keep POST or do we need GET as a method?

Reply:
commented on other discu https://github.com/symfony/symfony/pull/45185#discussion_r792755633
with `query` and `POST` it works well